src/components/DenoByExample/ByExample.tsx

```diff
@@ -0,0 +1,248 @@
+import React from "react";
+import Layout from "@theme/Layout";
+import Footer from "@theme/Footer";
+import { parseExample, DIFFICULTIES, TAGS, ExampleSnippet } from '../../plugins/deno-by-example/example';
+import CodeBlock from "@theme/CodeBlock";
+
+export default function ByExample({ example, examplesList }) {
+  const { name, content } = example;
+
+  const parsed = parseExample(name, content);
+
+  const {
+    id,
+    title,
+    description,
+    difficulty,
+    tags,
+    additionalResources,
+    run,
+    playground,
+    files,
+  } = parsed;
+
+  const freshProps = {
+    url: {
+      origin: "https://github.com/denoland/deno-docs/blob/by-example/",
```

Comment:
It seems the branch name is needed before the path?

```suggestion
      origin: "https://github.com/denoland/deno-docs/blob/main/by-example/",
```